src/Modules/ModulesGraded.jl

```diff
@@ -755,6 +755,56 @@ function set_grading(f::FreeModuleHom{T1, T2}) where {T1 <: FreeMod_dec, T2 <: F
 end
 # for decorations: add SubquoModule_dec for codomain once it exists
 
+@doc raw"""
+    degree(a::FreeModuleHom)
+
+If `a` is graded, return the degree of `a`.
```

Comment:
Maybe phrase it as 'return the degree of a graded FreeModuleHom a'.
(The way you are phrasing it, immediately leads to the question 'and otherwise?')